wsmaster/che-core-api-workspace/src/main/java/org/eclipse/che/api/workspace/server/WorkspaceManager.java

```diff
@@ -585,6 +585,16 @@ private void handleStartupSuccess(String workspaceId) {
     }
   }
 
+  /**
+   * Gets total count of all workspaces
+   *
+   * @return workspaces count
+   * @throws ServerException when any error occurs
+   */
+  public long getTotalCount() throws ServerException {
```

Comment:
IMHO `getWorkspaceCount()` would be a better name.

Reply:
I don't mind. I just want to make it looks similar to UserManager.  @mshaposhnik @sparkoo  @sleshchenko  wdyt?

Reply:
I'm for consistency here. Or shorten it to `getCount()` or even just `count()` as word `total` or doubling word `workspace` imho does not add any descriptive value.

Reply:
I'll just leave this here then :smile: `createWorkspace()`, `updateWorkspace()`, `startWorkspace()`, `stopWorkspace()`, `getWorkspace()`, `removeWorkspace()`..

For me, `WorkspaceManager.getTotalCount()` just doesn't tell what that total count is of.

But it is a nitpick, if you prefer `getTotalCount()` then so be it, I don't mind too much.

Reply:
`getTotalWorkspaceCount` ?